config/openshift/webhook.yaml

```diff
@@ -16,9 +16,37 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: tekton-operator-webhook
```

Comment:
as the controller deployment is renamed to `openshift-` do we waana rename this also?

Reply:
we can do that.. Let me handle this in a follow up pr where i configure the config.yaml for openshift.